wikimedia/wikipedia-ios
# Talk Pages - Handle cell links

**Phabricator:** 
https://phabricator.wikimedia.org/T310275

### Notes
This PR turns the talk page html (title, lead reply, other replies) into attributed strings, and routes any link taps within those strings. Since we are actively working on the sticks in these cells, I tried to keep the changes low-touch in case we run into merge conflicts. Any spacing issues, etc can be looked into once the sticks are in place.

### Test Steps
1. Load talk page topic cells & reply threads.
2. Confirm you now see links in topic cells and reply threads. Confirm links can be tapped and the appropriate new screen is pushed on.
3. Confirm you no longer see html tags in the topic cells.
